### ISSUE TITLE
Plugin crashes app every time I try to export a page

### ISSUE THREAD

**Issue by andrewharwood-rev** [sketch-hq/print-export-sketchplugin#33]
**Steps to reproduce:**

1. Open sketch app
2. Install/enable "Print Export Plugin"
3. Navigate to page which you want to export.
4. In the menu bar, select "Plugins"-> "Print Export"
5. In the dialog box select select the following options:
— Sketch page per PDF Page
— Export current page)
— Show Artboard Shadow
— Show Artboard Name
— Show Prototyping Links
— Paper Size Standard, US
— Page Size, Tabloid
— Landscape
6. Click Export
7. Name new file with system dialog.

**Expected Behavior**
PDF file is created, & saved to location specified.

**Actual Behavior**
Sketch App Crashes

This behavior can be consistently reproduced.

On a related note, I suspect that the error has something to do with the size, content, or complexity of my sketch file.  I tried reproducing this error with a new sketch file with only a few empty artboards and the plugin worked as expected.

There is something about my specific file which is causing this problem.  Perhaps it's just too large?

Please prioritize this bug, I cannot use this plugin at all.

**Comment by zenonmilou** [comment]
@andrewharwood-rev Does it work if you disable 'Show Prototyping Links', as it does for me?

**Comment by andrewharwood-rev** [comment]
@zedman ,

Yes the plugin does not crash if the "show prototyping links" box is left unchecked (see attachment).

This is really bad for me since I need the prototyping links to be displayed.

![Screen Shot 2020-08-10 at 2 44 49 PM](https://user-images.githubusercontent.com/16582236/89824248-6f3a2100-db18-11ea-8f0c-ab6450d9ad5b.png)

**Comment by ericcheungwork** [comment]
I experienced exactly the same as andrewharwood-rev. (Crash after click Export and no crash if uncheck "Show prototyping links")

I want to see the prototyping links in the pdf since it greatly facilitates the discussion in my team. 
Many thanks ><"

**Comment by shinji310** [comment]
I have been experiencing this issue.  I am on Version 68 (102490) with MacOS 10.15.6.  Hope that it gets fixed soon.  Or please share a walk around.
